models/public/midasnet/midasnet.md

```diff
@@ -0,0 +1,104 @@
+# midasnet
+
+## Use Case and High-Level Description
+
+MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+as described in the following paper:
+"Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+<https://arxiv.org/pdf/1907.01341>
+
+The model input is a blob that consists of a single image of "1x3x384x384" in `BGR` order.
```

Comment:
```suggestion
The model input is a blob that consists of a single image of "1x3x384x384" in `RGB` order.
```
Because it is description of original model.

Reply:
Original or converted model?
original: RGB
converted: BGR